resources/assets/less/bem/footer.less

```diff
@@ -24,13 +24,14 @@
     margin-bottom: 5px;
     display: flex;
     justify-content: center;
+    flex-wrap: wrap;
   }
 
   &__link {
     .link-plain();
     font-weight: bold;
     color: white;
-    margin: 0 10px;
+    margin: 5px 10px;
```

Comment:
this adds extra margin on normal pages and especially messes up footer on chat page

Reply:
I'll change it to mobile only